src/person.rb

```diff
@@ -7,12 +7,18 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It is expected that  `rentals`  instance variables should have Getters and Setters as you did for  `age` and `name`. Kindly fix that.

Reply:
Apply the same explanation for this case as well. the setter is defined as showed below:
![image](https://github.com/Diegogagan2587/school-library/assets/61764778/7082907e-76b7-4b1c-b7f0-1ce2640e3d83)